### PR TITLE
Change account flow so user id in analytics won't be empty

### PR DESCRIPTION
**Pull request by teodorus-nathaniel**
image

### CODE REVIEW

File: src/stores/my-account.ts [hunk 1]
```diff
@@ -13,50 +14,73 @@ type State = {
...
 
-const initialState = {
+const initialState: State = {
   isInitializedAddress: true,
   address: null,
   signer: null,
   energy: null,
-  unsubscribeEnergy: () => undefined,
   encodedSecretKey: null,
+  _currentSessionSecretKey: '',
```
Comment (olehmell) [inline src/stores/my-account.ts:50]: You use `_` before name for private methods, right?

**Review by olehmell — CHANGES_REQUESTED** [review]

File: src/stores/my-account.ts (hunk 1 above)
Comment (teodorus-nathaniel) [reply src/stores/my-account.ts:50]: yes

Comment (olehmell) [reply src/stores/my-account.ts:50]: Can I get it from other components?

**Review by olehmell — CHANGES_REQUESTED** [review]

**Comment by teodorus-nathaniel** [comment]
yes, all the states there are public, but I make the session private as I use it locally only in that store

**Review by olehmell — APPROVED** [review]

**Pull request merged**